src/mscorlib/ref/mscorlib.cs

```diff
@@ -7344,7 +7344,7 @@ public enum AssemblyNameFlags
     [System.Runtime.InteropServices.ComVisibleAttribute(true)]
     public class AssemblyNameProxy : System.MarshalByRefObject
     {
-        internal AssemblyNameProxy() { }
+        public AssemblyNameProxy() { }
```

Comment:
@jkotas To match Xamarin mscorlib, yes, we need to expose `GetAssemblyName` as well. However, that method calls a runtime method which doesn't have an implementation yet. We are tracking the list of not-yet-exposed APIs from `Reflection` [here](https://github.com/dotnet/corefx/issues/10530).